src/Mod/OpenSCAD/InitGui.py

```diff
@@ -47,11 +47,11 @@ def QT_TRANSLATE_NOOP(scope, text):
         import OpenSCAD_rc,OpenSCADCommands
         commands=['OpenSCAD_ReplaceObject','OpenSCAD_RemoveSubtree',\
             'OpenSCAD_RefineShapeFeature',\
-            'OpenSCAD_IncreaseToleranceFeature', 'OpenSCAD_Edgestofaces', \
+            'OpenSCAD_IncreaseToleranceFeature', 'OpenSCAD_Edgestofaces',\
             'OpenSCAD_ExpandPlacements','OpenSCAD_ExplodeGroup']
         toolbarcommands=['OpenSCAD_ReplaceObject','OpenSCAD_RemoveSubtree',\
-            'OpenSCAD_ExplodeGroup','OpenSCAD_RefineShapeFeature']
-            #'OpenSCAD_IncreaseToleranceFeature' #icon still missing
+            'OpenSCAD_ExplodeGroup','OpenSCAD_RefineShapeFeature',\
+            'OpenSCAD_IncreaseToleranceFeature']
```

Comment:
You can remove the backslash in all these. When the arguments are given inside parentheses or brackets, they don't need the backslash for line continuation. And they should preferably be visually aligned to the first element. This is,
```
toolbarcommands = ['OpenSCAD_ReplaceObject', 'OpenSCAD_RemoveSubtree',
                   'OpenSCAD_ExplodeGroup','OpenSCAD_RefineShapeFeature',
                   'OpenSCAD_IncreaseToleranceFeature']
```

Reply:
Done in 8bf68bd  
Thanks!